src/main/resources/avro/genotypephenotype.avdl

```diff
@@ -0,0 +1,138 @@
+@namespace("org.ga4gh")
+/**
+
+This protocol defines the associations between genotype
+and phenotype (G2P).  Associations can be made as a
+result of literature curation, computational modeling,
+inference, etc., and modeled and shared using this schema.
+
+Here, we follow the dogma of:
+
+      Genotype + Environment = Phenotype
+
+where a G2P association is between the G(enotype) in the context of
+some E(environment), which gives rise to a P(henotype). These
+associations have further evidence, provenance, and attribution.
+
+We leverage the GenomicFeature in the sequenceAnnotation schema here
+as it can accomodate any genomic feature from a single nucleotide variation
+(SNV), up through a gene, and/or complex rearrangements.  Each can
+be modeled as genomic features, and generally linked to a phenotype.
+Collections of these features can represent a genotype at different levels
+of completeness.  Therefore, we can represent single allelic variation,
+allelic complement, and multiple variants in a genotype that can each or
+collectively be associated with a phenotype.
+
+To enable standardized integration, this schema relies heavily on
```

Comment:
@heckerma good question.  I had not really considered epigenetic factors yet in the G2P schema, as I hadn't heard many folks discussing this with the GA4GH context yet (pardon me if i missed those conversations).  The sequence ontology captures different types of epigenetic markers as subclasses of [epigenetically_modified_region](http://sequenceontology.org/browser/current_svn/term/SO:0001720), so it is not difficult to see how those markers would fit into the <code>sequenceAnnotation</code> schema, and <code>GenomicFeatures</code> of those types could be added to the array of <code>GenomicFeatures</code>.  For the @monarch-initiative we have only started modeling epigenetic factors. I personally would consider epigenetic signatures as part of a genotype, as it is attached to the genome (though clearly could be altered due to external environmental perturbations).  That said, in Monarch we model genotypes to be comprised of two parts, intrinsic and extrinsic:  we include anything related to the DNA sequence as intrinsic and anything related to genome-targeting-reagents-resulting-in-altered-expression as extrinsic (such as addition of RNAi or morpholinos).  I can imagine that epigenetic markers would fit within a genotype, and probably the intrinsic genotype at that.  I will add a comment about being able to use <code>GenomicFeatures</code> for epigenetic markers, but we may want to discuss this in more detail in another git issue.